### ISSUE TITLE
TS8: System didn't automatically save unfinished work before session expiration

### ISSUE THREAD

**Issue by HannahCastro** [vasavah/vsv-fli-grn#49]
**Describe the bug**
System didn't automatically save unfinished work before session expired.

**Expected behavior**
As stated in our test case, system should save unfinished work before system automatically logs out of the account after 15 minutes idle time.

**Desktop (please complete the following information):**
 - OS: [Windows 7]
 - Browser [Google Chrome]
 - Version [75.0.3770.100]

**Comment by VCI-Jojo** [comment]
@HannahCastro Kindly validate if this is still the case on your end. Thank you.

**Comment by HannahCastro** [comment]
@VCI-Jojo Yes, this still occurs. Example is canvasser was creating JO, then was idle at this stage.
![image](https://user-images.githubusercontent.com/51902789/82518191-5fcabd80-9b51-11ea-890c-3b04735f50ec.png)
Session expires, and when user logs in again and checks PO they were processing, JO was not saved.
![image](https://user-images.githubusercontent.com/51902789/82518152-432e8580-9b51-11ea-871a-4ddfd7c72785.png)

**Comment by VCI-Jojo** [comment]
@HannahCastro I will have this rechecked. Thank you.
